### PR TITLE
IRIS-4550 Script to trigger Curation CMS Rank update

### PR DESCRIPTION
**Pull request by garthwebb**
Ticket:
https://wikia-inc.atlassian.net/browse/IRIS-4550

### CODE REVIEW

File: maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+use \Wikia\Logger\WikiaLogger;
+
+ini_set('display_errors', 'stderr');
+ini_set('error_reporting', E_ALL);
+
+require_once(dirname(__FILE__) . '/../../Maintenance.php');
+
+class updateCurationCMSTopicRanks extends Maintenance {
```
Comment (WikiaTech) [inline maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php:13]: MINOR Rename class "updateCurationCMSTopicRanks" to match the regular expression ^[A-Z][a-zA-Z0-9]*$. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS101)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+			[
+				'elapsedTime' => microtime( true ) - $startTime
+			]
+		);
+	}
+
+	private function logException( Exception $e ) {
+		WikiaLogger::instance()->error( self::EXCEPTION_MESSAGE,
+			[
+				'exception_message' => $e->getMessage(),
```
Comment (jsutterfield) [inline maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php:48]: This indentation is a little funky.

File: maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ini_set('display_errors', 'stderr');
+ini_set('error_reporting', E_ALL);
+
+require_once(dirname(__FILE__) . '/../../Maintenance.php');
+
+class updateCurationCMSTopicRanks extends Maintenance {
+
+	const SERVICE_NAME = "curation-cms";
+	const TIMEOUT = 60;
+	const NO_CONTENT = 204;
```
Comment (jsutterfield) [inline maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php]: Looks like this isn't being used.

**Review by jsutterfield — APPROVED** [review]
Couple minor comments, apart from that LGTM.

File: maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php (hunk 2 above)
Comment (garthwebb) [reply maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php:48]: Its standard indentation.  Plus, doesn't it look a little familiar? ;)

https://github.com/Wikia/app/blame/a66ef23f66883fa9d5cf6f1283a2a44111873a6a/maintenance/wikia/cronjobs/deleteOldOnSiteNotifications.php#L57-L63

This script is a copy-alter of deleteOldOnSiteNotifications.php

Comment (jsutterfield) [reply maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php:48]: I think it's easier to see in [this view](https://github.com/Wikia/app/blob/e046a1cb685297a10f7ed0cb2e10f0b17d27f541/maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php#L47-L50). Looks like the last tab before `exception` got updated to 4 spaces.

**Edit** I'll just update that quickly so we can move it along.

Comment (garthwebb) [reply maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php:48]: Hooray!

**Pull request merged**